### DISCUSSION TITLE
22000.318.34.1 credits

### DISCUSSION THREAD

**Discussion by comdlg32** [valinet/ExplorerPatcher#312] [General]
Hello, i am techsalt. I want to tell that my Github user is the @toiletflusher. Not @techsalt. If you don't believe me, look at the IM section in https://winclassic.boards.net/user/1679. I just want to say that is my Github username. I am not sure if this is the right place to put this here, but if you change the credits to have my Github username on it, thanks.

**Comment by valinet** [comment]
Hi

Apologies, didn't think to check there, anyway, I modified in all the places where I remember.

Thanks for pointing out, sorry again for the honest mistake.
